### DISCUSSION TITLE
What does these new Turbo Boost profiles with "at Guaranteed" mean?

### DISCUSSION THREAD

**Discussion by merrkry** [seerge/g-helper#916] [Q&A]
I have read about https://github.com/seerge/g-helper/discussions/492. However, in a recent update, new profiles ("Efficient at Guaranteed" and "Aggressive at Guaranteed") was introduced and I didn't find any documentation for them.

What are the differences between the profiles? And what is your recommendation?

**Comment by kuonji-arisu** [comment]
This depends on your CPU.

Here are some performance tests with different CPU boost settings. However, they may not produce the same results on your device.

https://www.reddit.com/r/MSILaptops/comments/d90kyt/processor_performance_boost_mode_tests/

**Reply by kuonji-arisu** [reply]
FYI, I'm using the "Efficient Aggressive:"

**Comment by envyska** [comment]
disabled = off (good for cool device)
enabled = boost clock step by step (good for lowering temp spike frequency)
agressive = boost clock to upper limitt directly (good for benchmarks/maximizing frames)
efficient = lower clock(turns off boost) faster after usage
at guaranteed = windows selects boosting level first instead of cpu itself (legal troll)
